x/sequencer/client/cli/tx_data_item.go

```diff
@@ -116,6 +118,14 @@ func createMsgDataItem(clientCtx client.Context, cmd *cobra.Command) (msg *types
 		msg.DataItem.Tags = append(msg.DataItem.Tags, bundlr.Tag(tag))
 	}
 
+	// Add nonce tag
+	sequence, err := getAccountSequence(clientCtx, msg, signer)
+	if err != nil {
+		return
+	}
+	tag := bundlr.Tag{Name: types.SequencerNonceTag, Value: strconv.FormatUint(sequence, 10)}
+	msg.DataItem.Tags = append(msg.DataItem.Tags, bundlr.Tag(tag))
```

Comment:
Also:
<img width="853" alt="image" src="https://github.com/warp-contracts/sequencer/assets/84311757/415362ac-4720-40df-b523-e056988c9f60">
(here and in line 118)

Reply:
> how will this work in case of the Warp SDK? The Warp SDK will always have to frist (before creating and signing the data-item) ask the cosmos network for the current nonce, right?

Unfortunately, yes. I don't know if we are creating a series of interactions somewhere, but if we are, the nonce can only be retrieved once.